Code/training_team8.py

```diff
@@ -47,6 +51,7 @@
 
 # %% -------------------------------------- Training Prep ----------------------------------------------------------
 model = densenet121(num_classes=14).to(device)
+model.avgpool = nn.AdaptiveAvgPool2d(1)
```

Comment:
sure thing if it helps, but i don't understand it atm. Comment/links to help explain should be included.